app/ui-react/packages/api/package.json

```diff
@@ -11,29 +11,34 @@
   "license": "MIT",
   "devDependencies": {
     "@babel/core": "^7.1.5",
-    "@testing-library/jest-dom": "^4.0.0",
     "@types/expect": "^24.3.0",
     "@types/file-saver": "^2.0.0",
     "@types/jest": "^24.0.11",
+    "@types/patternfly-react": "*",
```

Comment:
This shouldn't be needed?

Reply:
reverted dependencies and removed patternfly